AAEmu.UnitTests/Game/Core/Managers/AccessLevelManagerTests.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System.Collections.Generic;
+using System.Reflection;
+
+using AAEmu.Game.Core.Managers;
+using AAEmu.Game.Models;
+
+using Xunit;
+
+namespace AAEmu.UnitTests.Game.Core.Managers
+{
+    public class AccessLevelManagerTests
+    {
+        private readonly AccessLevelManager _manager;
+
+        public AccessLevelManagerTests()
+        {
+            _manager = AccessLevelManager.Instance;
+            ResetManagerState();
+            ResetAppConfiguration();
+        }
+
+        private void ResetManagerState()
+        {
+            var cmdField = typeof(AccessLevelManager).GetField("CMD",
+                BindingFlags.NonPublic | BindingFlags.Instance);
+            cmdField.SetValue(_manager, new List<Command>());
+        }
```

Comment:
You could maybe simplify this part by instantiating a new AccessLevelManager:
```suggestion
        public AccessLevelManagerTests()
        {
            _manager = new AccessLevelManager();
            ResetAppConfiguration();
        }
```

(I should probably have done the same with `ExperienceManagerTests` )